src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/restrictions/job/AbstractUserCauseRestriction.java

```diff
@@ -30,10 +30,15 @@
 import hudson.model.CauseAction;
 import hudson.model.Queue;
 import hudson.model.Run;
+
+import java.text.MessageFormat;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution.PlaceholderTask;;
```

Comment:
Why? Unused AFAICT.

Reply:
Resolved in 1e23e1db048b04334c4c81ff29edc38f0199cf3d